homeassistant/components/flexit_bacnet/binary_sensor.py

```diff
@@ -47,6 +47,10 @@ async def async_setup_entry(
     )
 
 
+# Coordinator is used to centralize the data updates
+PARALLEL_UPDATES = 0
+
+
```

Comment:
Please move code changes to a separate (preliminary or follow-up) PR.

Reply:
I have put the changes in a separate PR: https://github.com/home-assistant/core/pull/138514